src/api/app/components/workflow_run_row_component.rb

```diff
@@ -0,0 +1,72 @@
+class WorkflowRunRowComponent < ApplicationComponent
+  attr_reader :workflow_run
+
+  def initialize(workflow_run:)
+    super
+
+    @workflow_run = workflow_run
+  end
+
+  def hook_action
```

Comment:
_webhook_ is more accurate than only _hook_. This comment applies to all code with _hook_

```suggestion
  def webhook_action
```